entity-framework/core/querying/filters.md

```diff
@@ -39,6 +39,16 @@ The predicate expressions passed to the _HasQueryFilter_ calls will now automati
 > [!NOTE]
 > It is currently not possible to define multiple query filters on the same entity - only the last one will be applied. However, you can define a single filter with multiple conditions using the logical _AND_ operator ([`&&` in C#](https://docs.microsoft.com/dotnet/csharp/language-reference/operators/boolean-logical-operators#conditional-logical-and-operator-)).
 
+## Use of navigations
+
+Navigations can be used when defining global query filters. They are applied recursively - when navigations used in query filters are translated, query filters defined on referenced entities are also applied, potentially adding more navigations.
```

Comment:
Should I ditch the explanation what it means that we apply filters recursively? Sentence seems complicated to me, but it ties to the warning about cycles. Thoughts?

Reply:
Seems okay to me.